doc/conf.py

```diff
@@ -96,7 +96,6 @@
 # autosectionlabel
 autosectionlabel_prefix_document = True
 
-
```

Comment:
This some linting update? Is it required to pass CI? Or can it be removed from the PR?

Reply:
It was a simple line change while I was testing the build with the `setup` function commented out. That function is for generating JumpStart models, and it takes too long every time I build. After local testing is done, I uncommented and left that blank space change. Will try to fix it

Reply:
fixed it